dev/io.openliberty.checkpoint/src/io/openliberty/checkpoint/internal/CheckpointApplications.java

```diff
@@ -33,14 +31,14 @@
     @Activate
     public CheckpointApplications(@Reference Checkpoint checkpoint, BundleContext bc) {
         this.checkpoint = checkpoint;
-        this.doCheckpoint = "applications".equals(bc.getProperty("io.openliberty.checkpoint"));
+        this.doCheckpoint = "applications".equals(bc.getProperty(Checkpoint.CHECKPOINT_PROPERTY_NAME));
     }
 
     @Override
     public void check() {
         if (doCheckpoint) {
             try {
-                checkpoint.snapshot(Phase.APPLICATIONS, new File("snapshot"));
+                checkpoint.snapshot(Phase.APPLICATIONS);
             } catch (SnapshotFailed e) {
```

Comment:
Not for this PR. But we need an issue to track what should happen when a snapshot fails on the prepare side vs the restore side.  On the prepare side it will likely be during a container image build, we need to ensure an image build can fail when this happens.  On the restore side It may be sufficient to FFDC here, but I suspect we will also want some translated messages for the logs on both prepare and restore here.  We should probably create a utility that can do the exception handling for both phases (features and applications).